include/openthread.h

```diff
@@ -248,6 +248,26 @@ ThreadError otActiveScan(uint32_t aScanChannels, uint16_t aScanDuration, otHandl
 bool otActiveScanInProgress(void);
 
 /**
+ * This function starts a Thread Discovery scan.
+ *
+ * @param[in]  aScanChannels  A bit vector indicating which channels to scan (e.g. OT_CHANNEL_11_MASK).
+ * @param[in]  aScanDuration  The time in milliseconds to spend scanning each channel.
+ * @param[in]  aCallback      A pointer to a function that is called when a beacon is received or the scan completes.
+ *
+ * @retval kThreadError_None  Accepted the Thread Discovery request.
+ * @retval kThreadError_Busy  Already performing an Thread Discovery.
+ *
+ */
+ThreadError otDiscover(uint32_t aScanChannels, uint16_t aScanDuration, otHandleActiveScanResult aCallback);
+
+/**
+ * This function determines if an IEEE 802.15.4 Thread Discovery is currently in progress.
```

Comment:
whether we can replace "IEEE 802.15.4" with "MLE"?  just for distinguishing with 15.4 Active Scan command?